### PR TITLE
Add Carthage compatibility badge to the README

### PR DESCRIPTION
**Pull request by iCodesign**
Hope it will help. #1810

### CODE REVIEW

**Comment by jspahrsummers** [comment]
Thanks! Is it possible to include this in the header line? :pray:

**Comment by iCodesign** [comment]
Done!

**Comment by jspahrsummers** [comment]
@iCodesign Thanks!
@ReactiveCocoa/reactivecocoa :+1:?

**Comment by robrix** [comment]
:+1:

**Comment by jspahrsummers** [comment]
:sparkles::shipit:

File: Carthage/Checkouts/xcconfigs [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit b09b4b63235b760a3a48a8dd19c0415aaaa8f269
+Subproject commit 2e77204b59c3d97c24e5dd34966fb32c231194f0
```
Comment (jspahrsummers) [inline Carthage/Checkouts/xcconfigs]: Missed this before merging. Reverted in 0bb278e.

**Pull request merged**